prody/dynamics/heatmapper.py

```diff
@@ -131,7 +131,11 @@ def parseHeatmap(heatmap, **kwargs):
             numbers.append(items[:nnums])
         else:
             items = [arr]
-        heatmap.append(fromstring(items[-1], float, sep=';'))
+            
+        try:
+            heatmap.append(fromstring(items[-1], float, sep=';'))
+        except:
+            heatmap.append(frombuffer(items[-1], float, sep=';'))
```

Comment:
These fixes might be too ad-hoc. There should be a better way to determine which function to use. In addition, it is better the catch the specific error than a general "except"